gpu-jobs.md

```diff
@@ -34,15 +34,26 @@ capabilities (as of 3/16/2020):
 
 ## A. General Use GPUs via the CHTC GPU Lab
 
-There are a limited number of shared use GPUs available through the CHTC GPU
-Lab. Therefore, these servers have different policies about job runtimes and
-the maximum number of running jobs per user than general CHTC servers.
-These GPUs are a special investment from the UW2020 program, and the policies
-aim to maximize how many researchers can benefit from this investment.
-
-By opting-in to use the CHTC GPU Lab servers, you agree to be contacted by the
-project leaders occasionally to discuss your GPU computing and help improve the
-GPU Lab.
+CHTC has a set of GPUs that are available for use by any CHTC user with an 
+account on our high throughput computing (HTC) system. These shared use 
+GPUs form the [CHTC GPU Lab](/gpu-lab.shtml). 
+
+Because the CHTC GPU Lab is limited in size, and because the majority of 
```

Comment:
```suggestion
Because the CHTC GPU Lab was
funded by a special investment from the [UW2020 program](https://research.wisc.edu/funding/uw2020/round-5-projects/enabling-graphics-processing-unit-based-data-science/), we
```